src/Sensor/SensorBms.cpp

```diff
@@ -0,0 +1,144 @@
+#include "SensorBms.h"
```

Comment:
Can we develop a more clear naming structure for the various "CAN sensors"? Maybe we can call each one "SensorCan..." so this one would be "SensorCanBms.cpp" and CanListenerAccesories could become "SensorCanAccessories"? What do you think of that? Whatever you decide, update the CanListenerAccessories so it follows the same naming structure.

Reply:
These classes are different enough from the other sensors that it might make sense for their names to differ a little bit too?  CanSensor or CanListener classes as opposed to SensorCan?

Reply:
I agree. I do like Colby's naming of CanXXXX as it helps differentiate it from the other sensor namings not using CAN